### PR TITLE
Update build-in IO list with AMQP

### PR DESCRIPTION
**Pull request by jbonofre**
Update the build-in IO list with AMQP IO.

### CODE REVIEW

**Comment by jbonofre** [comment]
R: @iemejia 
CC: @davorbonaci

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/543/

Jenkins built the site at commit id 0dc49befcd6bee579fd1b36920da0df5ca2eedc8 with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/264/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/544/

Jenkins built the site at commit id 5a324e873ad402e7902bbbca2fc1b34127e853fa with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/264/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by iemejia** [comment]
LGTM, Merging in minutes..

**Comment by iemejia** [comment]
Mmm I could not merge it, it seems the beam-site does not exist anymore at apache, probably this is related to the move to gitbox. We need to update the instructions in the contribution guide with this and since I wasn't part of this discussion I am not sure where should I push now, so I prefer to wait until I know.
You can also self-merge if you know the new procedure. Thanks and sorry.

**Comment by jbonofre** [comment]
Yes, `beam-site` is now on gitbox: https://gitbox.apache.org/repos/asf?p=beam-site.git;a=summary

You have to update your git remote.

**Comment by jbonofre** [comment]
I'm checking why Jenkins is complaining about the test website.

**Comment by iemejia** [comment]
If I understood correctly the advantage of the move to gitbox was to merge the PR directly to github not to apache, so I am not sure if I should better put the remote to the github address, or am I wrong?

**Comment by jbonofre** [comment]
Yeah, you are right. I think we can also directly merge on gitbox too.

File: src/documentation/io/built-in.md [hunk 1]
```diff
@@ -95,6 +92,10 @@ This table contains I/O transforms that are currently planned or in-progress. St
     <td><a href="https://issues.apache.org/jira/browse/BEAM-1893">BEAM-1893</a></td>
   </tr>
   <tr>
+    <td>InfluxDB</td><td>Java<td>
```
Comment (iemejia) [inline src/documentation/io/built-in.md]: The closing <td> is missing an slash, it should be </td>

**Comment by iemejia** [comment]
I just put a comment of a minor fix, I already tried to do an to push directly to github but I got a protected branch hook declined warning and the push was stopped :S

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/548/

Jenkins built the site at commit id 62aa2f17435c2e29af970958947ed2584c3ac877 with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/264/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by iemejia** [comment]
@asfgit merge

**Pull request closed without merging**